src/locales/cy-GB.json

```diff
@@ -1,148 +1,154 @@
+  "$nextPage": "Tudalen Nesaf",
+  "$of": "o {0}",
+  "$singlePageView": "Gweld un dudalen ar y tro",
+  "$searchByPageNumber": "Gweld un dudalen ar y tro",
+  "$previous": "Blaenorol",
+  "$previousImage": "Delwedd Flaenorol",
+  "$previousPage": "Tudalen Flaenorol",
+  "$settings": "Dewisiadau",
+  "$twoPageView": "Gweld dwy dudalen ar y tro",
+  "$goHome": "Gweld y maint gwreiddiol",
+  "$imageUnavailable": "Nid oes delwedd ar gael",
+  "$rotateRight": "Cylchdroi i'r dde",
+  "$zoomIn": "Closio",
+  "$zoomOut": "Gwrthglosio",
+  "$adjustImage": "Addasu delwedd",
+  "$clearSearch": "Gwaredu",
+  "$displaying_A_of_B": "{0} {1} o {2}",
+  "$enterKeyword": "Mewnosodwch allweddair",
+  "$imageLowercase": "delwedd",
+  "$instanceFound": "Dangos 1 canlyniad ar gyfer '{0}' ",
+  "$instancesFound": "Dangos {0} canlyniad o '{1}' ",
+  "$nextResult": "Canlyniad nesaf",
+  "$pageLowercase": "tudalen",
+  "$previousResult": "Canlyniad blaenorol",
+  "$print": "Argraffu",
+  "$resultFoundFor": "canlyniad a geir ar gyfer",
+  "$resultsFoundFor": "canlyniad ar gyfer",
+  "$searchWithinItem": "Chwilio tu fewn i:",
+  "$locale": "Iaith",
+  "$navigatorEnabled": "Ffenest llywio ar gae",
+  "$clickToZoomEnabled": "Cliciwch gyda'r llygoden i glosio",
+  "$reducedMotion": "Lleihau symudiad (analluogi animeiddiadau)",
+  "$preserveViewport": "Cadw'r lefel o glosio",
+  "$uvWebsite": "<a href='https://github.com/universalviewer/universalviewer'>mwy o wybodaeth</a>",
+  "$custom": "dewisol",
+  "$embedInstructions": "Er mwyn ymgorffori'r eitem hon ar eich gwefan, cop\u00efwch a gosodwch y cod isod.",
+  "$height": "Hyd",
+  "$iiifManifest": "Maniffest IIIF",
+  "$shareInstructions": "Er mwyn rhannu'r eitem hon, cop\u00efwch y URL isod.",
+  "$size": "Maint:",
+  "$width": "Lled",
+  "$cancel": "Canslo",
+  "$confirm": "Cadarnhau",
+  "$viewTerms": "Darllenwch y Telerau a'r Amodau Llawn",
+  "$login": "Mewngofnodi",
+  "$log_in": "Mewngofnodi",
+  "$logout": "Allgofnodi",
+  "$authCORSError": "Mae angen diweddaru'ch porwr er mwyn galluogi rhannu adnoddau traws-safle.",
+  "$authorisationFailedMessage": "Ni lwyddoch i fewngofnodi. Rhowch gynnig arni eto.",
+  "$canvasIndexOutOfRange": "Canfas yn rhy fawr i'w arddangos (ceisiwch eto)",
+  "$fallbackDegradedLabel": "Mewngofnodi",
+  "$fallbackDegradedMessage": "Mewngofnodwch i weld y cynnwys hwn gyda phenderfyniad llawn.",
+  "$forbiddenResourceMessage": "Nid yw'ch hawliau mynediad presennol yn ddigonol i weld y ddelwedd hon",
+  "$mediaViewer": "Gwyliwr Cyfryngau",
+  "$skipToDownload": "Neidio i lawrlwythiadau a fformatau amgen",
+  "$pleaseEnterValue": "Mewnosodwch gwerth",
+  "$pleaseLogInToViewAtFullQuality": "Mewngofnodwch i weld yn ansawdd llawn.",
+  "$vr": "VR",
+  "$currentTime": "Amser Presennol",
+  "$duration": "Hyd",
+  "$mute": "Mudo",
+  "$pause": "Seibiant",
+  "$play": "Chwarae",
+  "$brightness": "Disgleirdeb",
+  "$contrast": "Cyferbyniad",
+  "$saturation": "Dirlawnder",
+  "$reset": "Reset",
```

Comment:
It looks like the "Reset" and "Remember my settings" strings need to be translated for the various languages. I think if translations are not available, it may be better to omit them than to put English text here, but I'm not 100% sure of that -- I'm not too familiar with this project's i18n workflows.